### ISSUE TITLE
Overwriting prompts

### ISSUE THREAD

**Issue by tiagoporto** [yeoman/yeoman-assert#26]
My test files is
```
describe('Package.json File', function () {
  before(function () {
    return yoTest
      .run(path.join(__dirname, '../../app'))
      .withPrompts({
        integrations: ['coveralls']
      })
      .toPromise()
  })

  it('With coveralls', function () {
    assert.equalsFileContent('package.json', fs.readFileSync(path.resolve(__dirname, 'coveralls.json'), 'utf-8'))
  })
})

describe('Package.json File', function () {
  before(function () {
    return yoTest
      .run(path.join(__dirname, '../../app'))
      .withPrompts({
        integrations: ['travis']
      })
      .toPromise()
  })

  it('With travis', function () {
    assert.equalsFileContent('package.json', fs.readFileSync(path.resolve(__dirname, 'travis.json'), 'utf-8'))
  })
})
```

On the first test `With coveralls` run with  parameter `integrations: ['coveralls']`, on the second I change the parameter `integrations: ['travis']`. The problem is, on the second test the first parameter still there.

Im doing something wrong?

**Comment by SBoudrias** [comment]
Nothing stands out from your example. Any chance that's a code bug on your side? This is a pretty common use case and is working fine on my generators.

**Comment by tiagoporto** [comment]
I've debug the code and the parameters are right. When I run `yo` manually the result is expected. But when I run the tests seems that the result are concatenated on the return of the before. There is a way to clean the variable `var yoTest = require('yeoman-test')` before execute another run()?

Now I don't know if the bug is on `yeoman-assert` or `yeoman-test`.

Thanks

**Comment by JoshuaKGoldberg** [comment]
> This is a pretty common use case and is working fine on my generators.

Indeed, I don't reproduce any issues locally with the latest versions of everything. Plus it's been >6 years since last activity on this issue. I'm going to go ahead and close this issue out as aged away. But if anybody else is seeing it, please do post a full reproduction (https://antfu.me/posts/why-reproductions-are-required). We'd be happy to take a deeper look. Cheers! 🎩
